integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWDTModelNoServer.java

```diff
@@ -0,0 +1,344 @@
+    for (int j = 1; j <= replicaCount; j++) {
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
+    logger.info("Domain {0} is fully started in namespace {1} - servers are running and application is available",
+        domainUid, domainNamespace);
+
+    // delete domain and cluster
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
+    deleteDomainResource(domainNamespace, domainUid);
+    deleteClusterCustomResource(clusterName, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is AdminServerName set and the named server is not in Server section.
+   * Verify WDT will create the named server as in the AdminServerName.
+   */
+  @Test
+  @DisplayName("Test in the WDT model file there is AdminServerName set and the named server is not in Server section")
+  void testWdtModelAdminServerNameSetNamedServerNotInServer() {
+
+    String wdtModelFile = "model-hasadminservername-namedservernotinserver.yaml";
+    imageName = createAndVerifyDomainImage(wdtModelFile);
+    String namedAdminServerPodName = domainUid + "-new-admin-server";
+
+    createMiiDomainAndVerify(domainNamespace, domainUid, imageName, namedAdminServerPodName,
+        managedServerPrefix, replicaCount);
+
+    // check that there is another pod domainuid-admin-server running in 7001 since it is defined in Server section
+    checkPodReadyAndServiceExists(domainUid + "-admin-server", domainUid, domainNamespace);
+
+    //check and wait for the application to be accessible in all server pods
+    for (int j = 1; j <= replicaCount; j++) {
+      String managedServerPodName = managedServerPrefix + j;
+      String expectedStr = "Hello World, you have reached server " + MANAGED_SERVER_NAME_BASE + j;
+
+      logger.info("Checking that application is running on managed server pod {0}  in namespace {1} with "
+          + "expectedString {3}", managedServerPodName, domainNamespace, expectedStr);
+      checkAppIsRunning(
+          domainNamespace,
+          managedServerPodName,
+          expectedStr);
+    }
+
+    logger.info("Domain {0} is fully started in namespace {1} - servers are running and application is available",
+        domainUid, domainNamespace);
+
+    // delete domain and cluster
+    shutdownDomainAndVerify(domainNamespace, domainUid, replicaCount);
+    deleteDomainResource(domainNamespace, domainUid);
+    deleteClusterCustomResource(clusterName, domainNamespace);
+  }
+
+  /**
+   * Test in the WDT model file there is configured cluster defined but no managed server associated with it in Server.
```

Comment:
Why validation Error.  I think it is a valid usecase in WebLogic. Can u try it out.  User can add servers later to the cluster.

Reply:
sent a msg to Johnny about this test case, waiting for his response.

Reply:
Discussed with Johnny. This is the behavior since day one and it is consistent with all type of domains.